src/Editor/Editor.jsx

```diff
@@ -34,17 +35,37 @@ class Editor extends Component {
   };
 
   render() {
-    
+
     return (
-      <div style={{width:'75%'}}>
-        <CodeMirror
+      <div style={{width:'75%',margin:'4%'}}>
```

Comment:
These are static option. Can't attach a click listener to each option. Make component which takes in a list and render the list as option with onClickListener attached to them.

Reply:
Didn't get that...

Reply:
You have hardcoded the option in the select tag, instead make a component which takes in a list of string and render that list dynamically. Anyways i have done it here [Selector Component](https://github.com/dharan1011/algo-run/blob/ui/src/Selector/Selector.jsx)